### PR TITLE
Latest reaper

### PR DESCRIPTION
**Pull request by ryansanford**
Update both docker env and bin/run.sh to use the latest reaper in the master branch of scitran/reaper as requested by @gsfr.

@nagem and @kofalt Please review and test.

### CODE REVIEW

File: bin/run.sh [hunk 1]
```diff
@@ -154,7 +155,7 @@ trap "{
 
 
 # Wait for everything to come up
-sleep 1
+sleep 5
```
Comment (kofalt) [inline bin/run.sh]: I take it this is uwsgi or someone being sluggish?

**Comment by kofalt** [comment]
LGTM dry-reviewed, will test when I get in.

**Comment by gsfr** [comment]
This appears to be competing with #236. Why?

**Comment by ryansanford** [comment]
I hadn't seen #236 -- my bad. I'll close this PR, and transition tweaks to that one...

**Pull request closed without merging**